modules/compliance-operator-install.adoc

```diff
@@ -8,9 +8,14 @@
 [role="_abstract"]
 Install the Compliance Operator using Operator Hub.
 
-.Procedure
+[IMPORTANT]
+====
+You must install the Compliance Operator before Sensor is fully operational. Otherwise, you need to restart Sensor in the secured cluster.
```

Comment:
```suggestion
If you install the Compliance Operator after Sensor is fully operational, you must restart Sensor in the secured cluster.
```

I'm afraid the way it was stated may lead customers to uninstall or something unnecessary because they didn't read the complete note.  But I could be overly paranoid.